src/components/forms/TextField/index.jsx

```diff
@@ -2,30 +2,33 @@
 import React from 'react'
 import MuiTextField, { TextFieldProps } from 'material-ui/TextField'
 
-const TextField = ({
-  input: {
-    name, onChange, value, ...restInput
-  },
-  meta,
-  render,
-  text,
-  ...rest
-}: TextFieldProps) => {
-  const helperText = value ? text : undefined
-  const showError = meta.touched && !meta.valid
+class TextField extends React.PureComponent<TextFieldProps> {
```

Comment:
@germartinez I have transformed this functional component to PureComponent class otherwise the ```const muiFields = TestUtils.scryRenderedComponentsWithType(open, TextField)``` does not work